docs/cilium.md

```diff
@@ -61,3 +61,53 @@ cilium_hubble_metrics:
 ```  
 
 [More](https://docs.cilium.io/en/v1.9/operations/metrics/#hubble-exported-metrics)
+
+## ip-masq-agent with Cilium
+
+If you are using masquerade: "true" and enable-bpf-masquerade: "true" together
+with tunnel: "vxlan" or "geneve"
+By default, any packet from a pod destined to an IP address outside of
+POD cidr range is masqueraded. To allow more fine-grained
+control, Cilium implements ip-masq-agent in eBPF which can be enabled
+setting "enable_ip_masq_agent" to true, this will add the necessary
+ConfigMap mounts to Cilium DS as optional and you will have to enable it with:
+
+```yml
+enable_ip_masq_agent: "true"
```

Comment:
This sample also needs to be updated to true, not "true"

Reply:
I've overlooked this, thanks! It's fixed.